### PR TITLE
feat(chat-tab): update input and button focus styles, add auto-scroll and hide scrollbar

### PR DESCRIPTION
**Pull request by eraykeskinmac**

### CODE REVIEW

File: src/components/chat-tab.tsx [hunk 1]
```diff
@@ -79,34 +80,41 @@ export const ChatTab = () => {
       : []),
   ];
 
+  useEffect(() => {
+    if (messagesEndRef.current) {
+      messagesEndRef.current.scrollIntoView({ behavior: "smooth" });
+    }
```
Comment (ytkimirti) [inline src/components/chat-tab.tsx]: Can we have this set to "instant", I like the effect but the scroll happening on every tab change is a bit annoying especially when there are lots of messages

**Comment by ytkimirti** [comment]
Looks good to me, thanks a lot for the PR, the auto scroll was needed.

Can you run
```shell
pnpm prettier -w src
```
so the formatting would be correct too

File: src/components/chat-tab.tsx (hunk 1 above)
Comment (eraykeskinmac) [reply src/components/chat-tab.tsx]: You are right i will add that

**Comment by eraykeskinmac** [comment]
@ytkimirti its done can you check

**Comment by ytkimirti** [comment]
Perfect, thanks a lot ❤️

**Pull request merged**